arrow-libs/core/arrow-core/src/commonTest/kotlin/arrow/core/continuations/NullableSpec.kt

```diff
@@ -111,4 +112,11 @@ class NullableSpec : StringSpec({
       string
     } shouldBe null
   }
+
+  "Either<Nothing, A> short circuit" {
```

Comment:
```suggestion
"Either<Nothing, A> can be bind"
```

`Either<Nothing, A>` can never result in `Left` and thus also never short-circuit.